scripts/godep-save

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+# make sure we are running in a toplevel directory
+if ! [[ -d '.git' ]]; then
+	echo "This script must be run in a toplevel rkt directory"
+	exit 255
+fi
+
+va=vendoredApps
+allApps=''
+if [[ -r "${va}" ]]; then
+	allApps=$(cat "${va}" \
+			 | grep -v '^[[:space:]]*\(//\|#\)' \
```

Comment:
Let's rewrite rkt in C. Death to the tax^Wgarbage collector! <grabs a torch and a pitchfork>

Anyway - merged two `grep -v` calls into one.